src/domain/task/command/createTask.ts

```diff
@@ -0,0 +1,17 @@
+import type { TaskStatus } from '../entity/task'
+
+export type CreateTask<I = string, T = string> = {
+  readonly id: I
+  readonly timestamp: Date
+  readonly type: T
+  readonly status: TaskStatus
+  readonly messageKey: string
+  readonly initiator?: string
+}
+
+export type AmendTask<I = string> = {
```

Comment:
Maybe we can put it in another file ? Or rename this one because it doesn't seems to fit very well in a `createTask.ts` file

Reply:
Good catch, 100% agree 😌